_posts/2017-10-02-learning-from-8thlight.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Learning from 8th Light, Day 1"
+date: 2017-10-02 10:03:00 +0200
+categories: how-we-learn 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 10
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-02-8thlight-1/8thlight.jpg
+---
+
+It all started at [BusConf], this August. Tobias Pflug (a colleague from HolidayCheck) asked [Daniel Irvine] (from [8th Light London]) if we could come over to London to learn from 8th Light about their apprenticeship program.
+Why? Because we want to be attractive to talent to stay competitive. And therefore we know we need to improve our skills continuously and keep producing high quality.
```

Comment:
maybe better "we want to attract talent and stay competitive".

Also it's best to not start sentences with "And" so you might want to change that somehow.